src/components/ui/datePicker.tsx

```
@@ -63,17 +189,17 @@ export const DatePicker: React.FC<DatePickerProps> = ({
-          <CalendarIcon className="mr-2 h-4 w-4" />
-          {date ? format(date, "PPP") : <span>Pick a date</span>}
+          <CalendarBlank className="mr-2 h-4 w-4" />
+          {getDatePickerLabel(date, mode)}
         </Button>
       </PopoverTrigger>
       <PopoverContent className="flex w-auto flex-col space-y-2 p-2">
-        {!!presets && (
+        {!!presets && mode === "single" && (
           <Select
             onValueChange={(value) => {
               const presetOffset = parseInt(value);
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)